recipes/cairosvg/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "CairoSVG" %}
+{% set version = "2.0.0rc6" %}
```

Comment:
RCs are fine in special cases. Can you write a note to why you need this particular `RC`? If that is not a "must" can you change to a stable release instead?